Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1727,15 +1718,14 @@ public virtual void LaunchUrl(string url)
                 throw new NotSupportedException();
             }
 
-            var process = ProcessFactory.Create(new ProcessOptions
+            var processStartInfo = new ProcessStartInfo
             {
                 FileName = url,
-                EnableRaisingEvents = true,
                 UseShellExecute = true,
                 ErrorDialog = false
-            });
-
-            process.Exited += ProcessExited;
+            };
+            var process = new Process { StartInfo = processStartInfo, EnableRaisingEvents = true };
+            process.Exited += (sender, args) => ((Process)sender).Dispose();
```

Comment:
Cannot we make a helper for that? (especially to make sure the `.Dispose()` thing is called)

Reply:
I wrote a helper for this, but then realised this is the only place this pattern is used so I don't think it's necessary.

The other instances just wrap the process in a `using` or have a more complicated `Exited` handler where this extension wouldn't apply